### PR TITLE
Feature/calendar events post sickleave

### PR DESCRIPTION
**Pull request by serjKim**

### CODE REVIEW

File: mobile/src/calendar/event-dialog/claim-sick-leave-event-dialog.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+class ClaimSickLeaveEventDialogImpl extends Component<ClaimSickLeaveEventDialogProps & ClaimSickLeaveEventDialogDispatchProps> {
+    public render() {
+        return <EventDialogBase 
+                    title={'Select date to Start your Sick Leave'} 
+                    text={this.text}
+                    icon={'sick_leave'} 
+                    cancelLabel={'Back'}
+                    acceptLabel={'Confirm'}
+                    acceptAction={this.acceptAction}
```
Comment (santee) [inline mobile/src/calendar/event-dialog/claim-sick-leave-event-dialog.tsx]: what about closing the dialog?

File: mobile/src/calendar/event-dialog/confirm-sick-leave-event-dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+class ConfirmSickLeaveEventDialogImpl extends Component<ClaimSickLeaveEventDialogProps & ClaimSickLeaveEventDialogDispatchProps> {
+    public render() {
+        return <EventDialogBase 
+                    title={'Select date to Complete your Sick Leave'} 
+                    text={this.text}
+                    icon={'sick_leave'} 
+                    cancelLabel={'Back'}
+                    acceptLabel={'Confirm'}
+                    acceptAction={this.acceptAction}
```
Comment (santee) [inline mobile/src/calendar/event-dialog/confirm-sick-leave-event-dialog.tsx]: maybe rename to onAcceptClick onCancelClick onCloseClick?

File: mobile/src/calendar/event-dialog/confirm-sick-leave-event-dialog.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    private acceptAction = () => {
+        this.props.confirmSickLeave(this.props.userEmployee.employeeId, this.props.startDay.date, this.props.endDay.date);
+    }
+
+    private closeDialog = () => {
+        this.props.closeDialog();
+    }
+
+    private get text(): string {
+        return `Your sick leave has started on ${this.props.startDay.date.format(eventDialogTextDateFormat)}${this.props.endDay ? ` and will be complete on ${this.props.endDay.date.format(eventDialogTextDateFormat)}` : ''}`;
```
Comment (santee) [inline mobile/src/calendar/event-dialog/confirm-sick-leave-event-dialog.tsx]: 'starts on' and 'completes on'.

File: mobile/src/calendar/event-dialog/claim-sick-leave-event-dialog.tsx (hunk 1 above)
Comment (serjKim) [reply mobile/src/calendar/event-dialog/claim-sick-leave-event-dialog.tsx]: it's optional

File: mobile/src/calendar/event-dialog/confirm-sick-leave-event-dialog.tsx (hunk 2 above)
Comment (serjKim) [reply mobile/src/calendar/event-dialog/confirm-sick-leave-event-dialog.tsx]: Press :)

**Pull request merged**